package.json

```diff
@@ -40,7 +40,7 @@
     "test:e2e": "yarn build:release && grunt test-e2e",
     "test:e2e:windows": "export SAUCE_PLATFORM_NAME=windows && yarn test:e2e",
     "test:testcafe-setup": "mkdir -p test-reports/testcafe && yarn start --mock mockDuo --open false",
-    "test:testcafe-run": "yarn test -t testcafe -c 2 chrome:headless --reporter xunit:build2/reports/junit/testcafe-results.xml",
```

Comment:
assume this is not intentional?

Reply:
Yes, reverted.
(I changed this temporary to see Testcafe errors in Travis)